components/sst_importer/src/import_mode.rs

```diff
@@ -120,6 +131,29 @@ impl ImportModeSwitcher {
         executor.spawn(timer_loop);
     }
 
+    // return whether some regions are in import mode
+    pub fn region_import_mode(&self) -> bool {
+        self.inner.lock().unwrap().import_mode_regions.is_some()
+    }
+
+    // set import mode for some regions
+    pub fn set_import_mode_regions(&self, import_mode_regions: HashSet<u64>) {
+        self.inner.lock().unwrap().import_mode_regions = Some(import_mode_regions);
+    }
+
+    pub fn clear_import_mode_regions(&self) {
+        self.inner.lock().unwrap().import_mode_regions.take();
+    }
+
+    // Should only be used in v2.
+    pub fn region_in_import_mode(&self, region_id: u64) -> bool {
```

Comment:
Check the self.ctx.importer.get_mode() in raftstore v1 and there're some logic may be ported to v2. 
For example, on_split_region_check_tick, it will skip if the region is under import mode. 
And for on_cleanup_import_sst, it's also used. We may need it in v2 as well.